sensor_test.py

```diff
@@ -1,25 +1,26 @@
 from conftest import wait
+from conftest import SensorInfo
 import logging
 
 log = logging.getLogger(__name__)
 
 def test_sanity(get_sensor_info, get_sensor_reading):
     sensor_info = get_sensor_info()
 
-    sensor_name = sensor_info.get("name")
+    sensor_name = sensor_info.name
     assert isinstance(sensor_name, str), "Sensor name is not a string"
 
-    sensor_hid = sensor_info.get("hid")
+    sensor_hid = sensor_info.hid
     assert isinstance(sensor_hid, str), "Sensor hid is not a string"
 
-    sensor_model = sensor_info.get("model")
+    sensor_model = sensor_info.model
     assert isinstance(sensor_model, str), "Sensor model is not a string"
 
-    sensor_firmware_version = sensor_info.get("firmware_version")
+    sensor_firmware_version = sensor_info.firmware_version
     assert isinstance(sensor_firmware_version, int), "Sensor firmware_version is not an int"
 
-    sensor_reading_interval = sensor_info.get("reading_interval")
-    assert isinstance(sensor_reading_interval, int), "Sensor reading_interval is not an int"
+    sensor_reading_interval = sensor_info.reading_interval
+    assert isinstance(sensor_reading_interval, int), "Sensor reading_interval is not a float"
```

Comment:
Sensor reading interval is an int, so there's no need to edit this message.

Reply:
![image](https://github.com/VictoriaSer/sensor/assets/163463048/0db21ebb-9652-4482-a7d7-595d1a96dc39)

there is an error meggase, that's why I removed get

Reply:
my mistake!
fixed the error message!